src/buffer/out/TextAttribute.hpp

```diff
@@ -169,6 +176,8 @@ class TextAttribute final
     TextColor _background;
     ExtendedAttributes _extendedAttrs;
 
+    USHORT _hyperlinkId;

```

Comment:
nit: prefer `unsigned short` in new code, or `uint16_t` (my preference) for explicit sizing. `ALL_CAPS_TYPE_NAMES` are typically seen in legacy Windows code